scripts/webpack/babel.config.js

```diff
@@ -1,53 +0,0 @@
-module.exports = function getBabelConfig(options = {}) {
-  const babelOptions = {
-    cacheDirectory: true,
-    cacheCompression: false,
-    babelrc: false,
-    // Note: order is bottom-to-top and/or right-to-left
-    presets: [
-      [
-        '@babel/preset-env',
-        {
-          bugfixes: true,
-          browserslistEnv: process.env.BABEL_ENV || options.BABEL_ENV || 'production',
-          useBuiltIns: 'entry',
-          corejs: '3.10',
-        },
-      ],
-      [
-        '@babel/preset-typescript',
-        {
-          allowNamespaces: true,
-          allowDeclareFields: true,
-        },
-      ],
-      [
-        '@babel/preset-react',
-        {
-          runtime: 'automatic',
-        },
-      ],
-    ],
-    plugins: [
-      [
-        '@babel/plugin-transform-typescript',
-        {
-          allowNamespaces: true,
-          allowDeclareFields: true,
-        },
-      ],
-      ['@babel/plugin-proposal-object-rest-spread', { loose: true }],
-      '@babel/plugin-transform-react-constant-elements',
-      '@babel/plugin-proposal-nullish-coalescing-operator',
-      '@babel/plugin-proposal-optional-chaining',
-      '@babel/plugin-syntax-dynamic-import', // needed for `() => import()` in routes.ts
-      'angularjs-annotate',
-    ],
-  };
-
-  if (options.REACT_REFRESH) {
-    babelOptions.plugins.push('react-refresh/babel');
-  }
```

Comment:
cc @jackw is it OK to remove it, at least for now?

Reply:
Yes whilst I'm currently working to get hot module reloading and fast refresh in a better state I don't believe anyone actually uses `start:hot` right now. Will work around these changes if/when the time comes.

Reply:
I ended up including this anyway just for parity, controlled through `NODE_ENV=hot`